src/main/java/ru/practicum/shareit/booking/BookingController.java

```diff
@@ -1,22 +1,76 @@
 package ru.practicum.shareit.booking;
 
+import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-import ru.practicum.shareit.booking.model.Booking;
+import ru.practicum.shareit.booking.dto.BookingCreateDto;
+import ru.practicum.shareit.booking.dto.BookingDto;
+import ru.practicum.shareit.booking.model.BookingState;
+import ru.practicum.shareit.item.UserIdHeader;
 
+import java.util.List;
+
+@Slf4j
 @RequiredArgsConstructor
 @Validated
 @RestController
 @RequestMapping(path = "/bookings")
 public class BookingController {
-    private final BookingService service;
+    private final BookingServiceImpl service;
+
+    @GetMapping("/{bookingId}")
+    public BookingDto getBookingById(@PathVariable long bookingId) {
+        return service.findById(bookingId);
+    }
+
+    @PostMapping
+    public BookingDto createBooking(
+            @RequestHeader(UserIdHeader.USER_ID_HEADER) Long userId,
+            @RequestBody @Valid BookingCreateDto dto) {
+        log.info("received dto" + dto);
```

Comment:
Стоит заменить конкатенацию строк на плейсхолдеры логгера - это более производительно и считается хорошей практикой. В текущей реализации строка `"received dto" + dto` будет вычисляться даже если уровень логирования INFO отключён.
```suggestion
        log.info("Received DTO: {}", dto);
```
Это обеспечит ленивую подстановку и улучшит читаемость лога.